cli/cmd/logs/logs.go

```diff
@@ -17,13 +17,14 @@ import (
 )
 
 type Logs struct {
-	F_Follow    bool   `desc:"Follow log output"`
-	S_Since     string `desc:"Logs since a certain time, either duration (5s, 2m, 3h) or RFC3339"`
-	P_Previous  bool   `desc:"Print the logs for the previous instance of the container in a pod if it exists"`
-	C_Container string `desc:"Print the logs of a specific container"`
-	R_Revision  string `desc:"Print the logs of a specific revision"`
-	N_Tail      int    `desc:"Number of recent lines of logs to print, -1 for all" default:"200"`
-	A_All       bool   `desc:"Include hidden or systems logs when logging"`
+	F_Follow            bool   `desc:"Follow log output"`
+	S_Since             string `desc:"Logs since a certain time, either duration (5s, 2m, 3h) or RFC3339"`
+	P_Previous          bool   `desc:"Print the logs for the previous instance of the container in a pod if it exists"`
+	C_Container         string `desc:"Print the logs of a specific container"`
+	R_Revision          string `desc:"Print the logs of a specific revision"`
+	N_Tail              int    `desc:"Number of recent lines of logs to print, -1 for all" default:"200"`
+	A_All               bool   `desc:"Include hidden or systems logs when logging"`
+	DT_DisableTimestamp bool   `desc:"Print the logs without timestamp"`
```

Comment:
What does the flag looks like after the change? the way it is structured right now is : A_BC so that long flag is BC and short flag is A. But I don't know if short flag allows two letters. Worth to check.

Reply:
```   --disable-timestamp, --dt    Print the logs without timestamp```

Open to suggestions on better naming too

Reply:
Ok. Looks good. Left to @ibuildthecloud for naming things. Personally I think it should be disabled by default?

Reply:
I would agree, think it's a lot of data and rather ugly.

Reply:
I'd probably just follow what docker does 
```
-t, --timestamps     Show timestamps
```

Reply:
so off by default.